jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java

```diff
@@ -601,11 +606,20 @@ public void setupDefaultActiveSearchFilters() {
     }
     
     public void openJobsView(final String pid) {
-        final PlaceRequest request = new DefaultPlaceRequest(JOBS);
-        if(pid != null){
-            request.addParameter(SEARCH_PARAMETER_PROCESS_INSTANCE_ID, pid);
-        }
-        placeManager.goTo(request);
+        navigateToPerspective(JOBS,
+                              SEARCH_PARAMETER_PROCESS_INSTANCE_ID,
+                              pid);
+    }
+
+    public void openTaskAdminView(final String pid) {
+        navigateToPerspective(TASKS_ADMIN,
+                              SEARCH_PARAMETER_PROCESS_INSTANCE_ID,
+                              pid);
+    }
```

Comment:
@bsremac added now. the rule is first try to navigate to tasks admin if the user has access to, othwerwhise the regular task view is used.